components/head.php

```diff
@@ -4,7 +4,7 @@
 		if ( $_SERVER['SCRIPT_NAME'] === '/donate.php' ) {
 			echo 'Miraheze | Donate';
 		} else {
-			echo 'Miraheze';
+			echo 'Miraheze - 100% ad-free, free wiki hosting';
```

Comment:
Should be capitalised for titles.
```suggestion
			echo 'Miraheze - Ad-free, Free Wiki Hosting';
```